dat/glsl/rounded_rect.frag

```diff
@@ -0,0 +1,24 @@
+#include "lib/sdf.glsl"
+uniform vec4 colour;
+uniform vec4 dimensions;
+uniform int parami;
```

Comment:
should probably be `uniform float paramf;` or the setting thickness part could give errors on some drivers.